src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/rishabhbohra3/Algo-tutorial-test/pull/4#issuecomment-2208447794>#aabba6</a><span> - 07/04/2024, 08:52 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Initializing 'absMax' to '0' instead of 'numbers[0]' can lead to incorrect results if all numbers in the array are negative. The initial value should be the first element of the array to correctly compare absolute values.
</div>
<div id="fix">
<b>Fix:</b> Initialize 'absMax' to 'Math.abs(numbers[0])' to ensure the correct initial value for comparison.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -16,7 +16,7 @@
  public static int getMaxValue(int... numbers) {
      if (numbers == null || numbers.length == 0) {
          throw new IllegalArgumentException("Numbers array cannot be empty or null");
      }
 -    int absMax = 0;
 +    int absMax = Math.abs(numbers[0]);
      for (int i = 1; i < numbers.length; i++) {
          if (Math.abs(numbers[i]) > absMax) {
              absMax = numbers[i];
          }
      }
      return Math.abs(absMax);
  }
```
</div>
</details>
</div>
<br>


</div>